esp8266/scripts/main.py

```diff
@@ -0,0 +1 @@
+# This script is run on boot
```

Comment:
It's still unclear what's the point of "running script on boot"? Does this try to emulate pyboard, which indeed runs boot.py, then possible main.py? But Python language doesn't do that.

(Note that I don't ask that to be removed or something, just for clarification how that's supposed to work/to be used).